api/notd/block_processor.py

```diff
@@ -59,12 +67,45 @@ async def get_transfers_in_block(self, blockNumber: int) -> List[RetrievedTokenT
         block = await self.ethClient.get_block(blockNumber)
         blockHash = block['hash'].hex()
         blockDate = datetime.datetime.fromtimestamp(block['timestamp'])
-        events = await self.ethClient.get_log_entries(startBlockNumber=blockNumber, endBlockNumber=blockNumber, topics=[self.erc721TansferEventSignatureHash])
-        logging.info(f'Found {len(events)} events in block #{blockNumber}')
-        allTokenTransfers = []
-        for eventsChunk in list_util.generate_chunks(events, 5):
-            allTokenTransfers += await asyncio.gather(*[self._process_event(event=dict(event), blockNumber=blockNumber, blockHash=blockHash, blockDate=blockDate) for event in eventsChunk])
-        return [tokenTransfer for tokenTransfer in allTokenTransfers if tokenTransfer]
+        #erc721events = await self.ethClient.get_log_entries(startBlockNumber=blockNumber, endBlockNumber=blockNumber, topics=[self.erc721TansferEventSignatureHash])
```

Comment:
why have you commented this out?

Reply:
ah i see just for testing. i would suggest you change this to have a big erc721 block and then a big erc1155 block i.e. move all the commented lines up here so all the erc721 stuff from this function is together and then all the erc1155 stuff is together just below that.